app/views/grantees/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="details-header">
+  <div class="grid-container">
+    <h1><%= @grantee.name %></h1>
+    <h2>Grantee #<%= @grantee.id %>, Region <%= @grantee.region %></h2>
+  </div>
+</div>
+<div class="details-body grid-container">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-4">
+      <div class="summary-section">
+        <div class="ct-section-divider">
+          <h2>Summary</h2>
+        </div>
+      </div>
+      <div class="grid-col-4">
```

Comment:
[suggestion] this div is unnecessary -- the Open Issues container is part of the `grid-col-4` div. by adding the other column, it is trying to divide itself smaller within the container grid i think.

if you remove this `div`, the Open Issues container should fill the whole column like the `summary` container.